tools/rosmsg/package.xml

```diff
@@ -25,6 +25,8 @@
   <run_depend>rosbag</run_depend>
   <run_depend>roslib</run_depend>
 
+  <test_depend>geometry_msgs</test_depend>
```

Comment:
In 1437c60 I deleted `test_rosmsgproto_geometry_py` and added to `test_rosmsgproto.py` tests of similar complexity to the deleted tests, but only using messages from `std_msgs`. The resulting test suite is not identical to the previous suite, but I don't see any significant changes in coverage.